plugins/s3-viewer-backend/src/service/router.ts

```diff
@@ -15,15 +15,19 @@
  */
 
 import {
-  PluginEndpointDiscovery,
+  // Remove this function as soon as all plugins support new LoggerService
```

Comment:
Please mark this (and the other comments) with `TODO` to ensure they don't get lost.

Reply:
I added the `TODO: ` to these 2 comments